types/react-jsonschema-form/index.d.ts

```diff
@@ -34,6 +35,7 @@ declare module 'react-jsonschema-form' {
         showErrorList?: boolean;
         ErrorList?: React.StatelessComponent<ErrorListProps>;
         validate?: (formData: T, errors: FormValidation) => FormValidation;
+        onBlur?: (...e: any[]) => any;
```

Comment:
This function should actually be typed like `WidgetProps['onBlur']`. On implementation you can actually spread the function parameters, but in typings you should declare the specific parameters. Rewrite this and make sure the tests reflect it.